Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG PARENT_VERSION=2.3.0-node20.15.0
+ARG PARENT_VERSION=2.5.0-node22.11.0	
```

Comment:
The other two you have done are using 2.8.4-node22.16.0, worth using the same here so they are kept in synch?